### DISCUSSION TITLE
What is `git.objects.base.Object.size`?

### DISCUSSION THREAD

**Discussion by CsatariGergely** [gitpython-developers/GitPython#1623] [Q&A] [answered]
git.objects.base.Object has a size property, but from the documentation it is not clear what is the unit of it.

**Comment by Byron** [comment, marked as answer]
That's true, and contributions are welcome if you know how to [document slots](https://github.com/gitpython-developers/GitPython/blob/1c8310d7cae144f74a671cbe17e51f63a830adbf/git/objects/base.py#L52).

The `size` field denotes the amount of bytes the object's data needs in memory when decoded.

**Comment by CsatariGergely** [comment]
Thanks for the answer. Can this be translated to something not related to the internal workings of Git?

**Reply by Byron** [reply]
I think if documentation was attached, the explanation could say it all and elaborately. But that's apparently not possible.
